apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -391,6 +443,14 @@ test('getRawDeviceStatus', async () => {
   expect(driver.getPaperHandlerStatus).toHaveBeenCalledTimes(1);
 });
 
+describe('paperHandlerStatusToEvent', () => {
+  test('PAPER_IN_OUTPUT', () => {
+    expect(paperHandlerStatusToEvent(getPaperInRearStatus())).toEqual({
+      type: 'PAPER_IN_OUTPUT',
+    });
+  });
+});
+
```

Comment:
Added for coverage. I couldn't find where branch coverage went down with my changes but this brings it back up.